base/ltfssdcl.dtx

```diff
@@ -507,8 +569,138 @@
  \expandafter#1\ifx\math@bgroup\bgroup{#4}\else#4\fi
  }%
 }
+%    \end{macrocode}
+%
+%
+
+%  \begin{macro}{\freeze@math@version}
+% \changes{v3.0x}{2021/09/26}{Macro added for (gh/676)}
+%    This command stores the current state of the math version and sets
+%    things up to return to it after each formula from now on. We use
+%    L3 programming layer code to set it up.
+%    \begin{macrocode}
+\ExplSyntaxOn
+\cs_new:Npn\freeze@math@version #1 {
+%    \end{macrocode}
+%    Save the current \cs{mv@\meta{version}} code and the number of
+%    allocated mathgroups inside.
+%    \begin{macrocode}
+  \@font@info{Freeze~ math~ alphabet~ allocation~ in~ version~ #1}
+  \cs_gset_eq:cc { mv@#1@frozen }{ mv@#1 }
+  \tl_gset:cx { g__mv_frozen_ #1 _tl }{ \int_use:c { c@mv@#1 } }
+%    \end{macrocode}
+%    Doing the reset the first time, we wait until we are out of
+%    math mode, so we use some recursive \cs{aftergroup} for this
+%    before we execute \cs{mv@\meta{version}@reset}.
+%    \begin{macrocode}
+  \aftergroup \__mv_init_freeze:N \exp_after:wN
```

Comment:
`\group_insert_after:N` instead of `\aftergroup`

Reply:
oversight, thought it was still only `:D`